lib/html_to_portable_text/index.ts

```diff
@@ -1,11 +1,12 @@
 import { htmlToBlocks } from "@sanity/block-tools"
+import type { PortableTextBlock } from "@portabletext/types"
 
 import { collibraRichText } from "./schemas/collibraRichText"
 import { rules } from "./rules"
 import { sanitizeHtml } from "../sanitize_html"
 import { htmlStringToDom } from "../html_string_to_dom"
 
-export const htmlToPortableText = (html: string) => {
+export const htmlToPortableText = (html: string): PortableTextBlock[] => {
   return htmlToBlocks(html, collibraRichText.get("content").fields.find((field) => field.name === "body").type, {
```

Comment:
I know this is not part of the PR, but the return statement here seems unnecessary

Reply:
it is, but in my eyes, it increases legibility

Reply:
I disagree. I think it would be more declarative and easier to read without the return.